tools/experiments/README.md

```diff
@@ -20,12 +20,25 @@ For content served from any other domain, you can toggle experiments in the Chro
 AMP.toggleExperiment('experiment')
 ```
 
+## Document opts in an experiment
+Document can choose to opt in a certain experiments. To do that, simply put a meta tag of name `amp-experiments-opt-in` in the head of the HTML document. Its `content` value is a comma separated string of experiment IDs to opt in.
+```HTML
+<head>
+  ...
+  <meta name="amp-experiments-opt-in" content="experiment-a,experiment-b">
+  ...
+</head>
+```
+By doing so, the specified experiments will be enabled for all visitors of the document.
+Note not all experiments allow document level opt-in. For a full list of whitelisted experiments, please see `allow-doc-opt-in` attribute in the project's `prod-config.json` file.  
+ 
 ## Adding a new experiment
 
 To add a new experiment:
 
-1.  Add the new experiment to this [list](https://github.com/ampproject/amphtml/blob/master/tools/experiments/experiments.js).
-2. Use it like this:
+1. Add the new experiment to this [list](https://github.com/ampproject/amphtml/blob/master/tools/experiments/experiments.js).
+1. Decide if the experiment should allow document level opt-in, add it to `allow-doc-opt-in` in `prod-config.json` and `canary-config.json` if so.
```

Comment:
True. It will be more common for amp-inabox if later we control some ads specific experiments at server side. Added one sentence guidance.

This is actually the preferred way to do numbering in MD, it generates numbers for you.